packages/plugins/tests/free-payment.test.js

```diff
@@ -0,0 +1,25 @@
+import {  PaymentFreePrice } from '../src/pricing/free-payment';
+
+
+describe("PaymentFreePrice", () => {
```

Comment:
I think it does not make sense to test a response of a no-argument function and dupplication of metadata. I mean when you change the properties for ex. increase the version this is going to fail and will lead to duplicate work.

I'd either:
- Remove the test file completely
- Actually test that free payment leads to 0.00 payment fee (= free) and not null/undefined/exceptions & remove the property test